### PR TITLE
Update README.md. Create Chinese README.

### PR DESCRIPTION
**Pull request by JimZhouZZY**
Changes:
+ Modified `README.md`
  * Added link to translated README at the beginning.
  * Added close menu function to`left click`
  * Added key `Esc`
+ Created `README` floder and `zh-cn.md` `zh-cht.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 [![latest release](https://img.shields.io/github/v/release/Grimmys/rpg_tactical_fantasy_game)](https://github.com/Grimmys/rpg_tactical_fantasy_game/releases/latest)
 ![GitHub release (latest by date)](https://img.shields.io/github/downloads/Grimmys/rpg_tactical_fantasy_game/latest/total)
 
+[ [English](README.md) | [简体中文](README/zh-cn.md) | [繁体中文](README/zh-cht.md) ]
```
Comment (Grimmys) [inline README.md:8]: 🤩

File: README.md [hunk 2]
```diff
@@ -48,8 +50,9 @@ Then you can run `python main.py` or "./main.py" (only for Python 3) in linux op
 ## Keys
 
 * Left click : Select a player, choose a case to move, select an action to do etc (main button)
-* Left click (on any empty tile) : Open main menu
+* Left click (on any empty tile) : Open or close main menu
 * Left click (on any entity that is not a player who has finished his turn) : Open a window giving information about the
   entity
 * Right click : Deselect a player or cancel last action if possible (secondary button)
 * Right click (on any entity) : Show the possible movements of the entity
+* Ecs key : Close a menu on the top layer
```
Comment (Grimmys) [inline README.md:58]: It's "Esc" not "Ecs"

**Review by Grimmys — CHANGES_REQUESTED** [review]
Looks good, thanks for the contribution!

Just a small typo to fix.

Also, for my personal knowledge, can you explain in a few words (or give me a ressource link) what's the difference between these two translation README files? Is it like different languages? Do you have people in China understanding one "language" but not the other for example?

File: README.md (hunk 2 above)
Comment (JimZhouZZY) [reply README.md:58]: Bruh.

**Comment by JimZhouZZY** [comment]
They are Chinese languages that used in different regions. Traditional Chinese (code `zh-cht`) is used in Hong Kong, MACAU and Tai Wan, now. Simplified Chinese is used in China mainland now. It is a simplified version of traditional Chinese developed in 20th century. The characters are mostly same but simplified Chinese has fewer strokes. For example “龟” and “龜”, it is obvious that the first one is simplified.

**Review by Grimmys — APPROVED** [review]

**Comment by Grimmys** [comment]
Thanks for giving me some context.

I knew about "simplified" chinese but not more.

**Pull request merged**